mat64/io_test.go

```diff
@@ -39,3 +49,36 @@ func TestDenseRW(t *testing.T) {
 		}
 	}
 }
+
+func TestVectorRW(t *testing.T) {
+	for i, test := range []*Vector{
+		NewVector(0, []float64{}),
```

Comment:
I don't think I/O should panic. I/O is a file error, it's not a programmer error. It is not a knowable error before runtime and it is possible to handle it, so it should be an error.